input/postgres/relations.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/pganalyze/collector/state"
 )
 
+const relationSQLdefaultSystemCatalogFilter = "n.nspname NOT IN ('pg_catalog', 'pg_toast', 'information_schema')"
+const relationSQLepasSystemCatalogFilter = "n.nspname NOT IN ('pg_catalog', 'pg_toast', 'information_schema', 'sys') AND n.nspparent <> 'sys'::regnamespace"
```

Comment:
Note that "nspparent" is not a feature in regular Postgres (in EPAS there are a handful of internal schemas that all are children of "sys").

Reply:
not a strong opinion but it can be all capital letters as it's an acronym:

```suggestion
const relationSQLEPASSystemCatalogFilter = "n.nspname NOT IN ('pg_catalog', 'pg_toast', 'information_schema', 'sys') AND n.nspparent <> 'sys'::regnamespace"
```